gemfiles/jruby_9.2.0.0_contrib.gemfile

```diff
@@ -0,0 +1,78 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
```

Comment:
It's a bit of a shame that Appraisal doesn't seem to support the new `gems.rb` name for `Gemfile`s. Because it ends in `.rb`, it's not a "magic" name anymore so we'd get syntax highlighting :|